queue_job/data/queue_data.xml

```diff
@@ -2,6 +2,16 @@
 <odoo>
     <data noupdate="1">
 
+        <record id="ir_cron_queue_job_garbage_collector" model="ir.cron">
+            <field name="name">Jobs Garbage Collector</field>
+            <field name="interval_number">5</field>
+            <field name="interval_type">minutes</field>
+            <field name="numbercall">-1</field>
+            <field eval="'queue.job'" name="model"/>
+            <field eval="'requeue_stuck_jobs'" name="function"/>
+            <field eval="'()'" name="args"/>
```

Comment:
The eval's is not necessary. You can just put the strings as the content of the tags.